appveyor.yml

```diff
@@ -0,0 +1,94 @@
+environment: 
+    matrix:
+        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+          DB: 10.3.11
```

Comment:
Does it have to be concrete version? Will we always need to update the version here?

Reply:
Yes, that is a problem, but I didn't yet figure out how to do it differently